### PR TITLE
I 12575 INT Playwright SIT test failure fix

### PR DESCRIPTION
**Pull request by r-mettler**
## Summary
Fix for failing playwright test. Use days instead of months.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the
2. Login as a
3.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Review by danieljordan-caci — APPROVED** [review]

File: pkg/testdatagen/testharness/make_move.go [hunk 1]
```diff
@@ -3946,8 +3946,8 @@ func MakeHHGMoveInSIT(appCtx appcontext.AppContext) models.Move {
 		},
 	}, nil)
 
-	twoMonthsAgo := now.AddDate(0, -2, 0)
-	oneMonthAgo := now.AddDate(0, -1, 0)
+	twoMonthsAgo := now.AddDate(0, -2, -60)
```
Comment (taeJungCaci) [inline pkg/testdatagen/testharness/make_move.go:3949]: Wouldn't this actually make it -2 months - 60 days = 120 something days?

Comment (taeJungCaci) [reply pkg/testdatagen/testharness/make_move.go:3949]: Should it be `twoMonthsAgo := now.AddDate(0, 0, -60)`?

Comment (cameroncaci) [reply pkg/testdatagen/testharness/make_move.go]: ^ Joining this comment. (0, -2, -60) is in fact -120

**Review by taeJungCaci — APPROVED** [review]

File: pkg/testdatagen/testharness/make_move.go (hunk 1 above)
Comment (r-mettler) [reply pkg/testdatagen/testharness/make_move.go]: Fixed

**Review by cameroncaci — APPROVED** [review]

**Review by deandreJones — APPROVED** [review]

**Pull request merged**